geometry/grassmann_test.cpp

```diff
@@ -88,6 +92,66 @@ class GrassmannTest : public testing::Test {
 
 using GrassmannDeathTest = GrassmannTest;
 
+TEST_F(GrassmannTest, VectorFMA) {
+  if (!CanEmitFMAInstructions || !HasCPUFeatures(CPUFeatureFlags::FMA)) {
```

Comment:
Skip the test in the fixture, https://google.github.io/googletest/advanced.html#skipping-test-execution.